public/app/core/services/backend_srv.ts

```diff
@@ -66,6 +66,11 @@ export class BackendSrv {
     var requestIsLocal = options.url.indexOf('/') === 0;
     var firstAttempt = options.retry === 0;
 
+    if (!options.url.match('https?://') && this.contextSrv && this.contextSrv.user && this.contextSrv.user.orgId) {
```

Comment:
what's the cause for this if case? the https chec specifically, are we using backendSrv for external calls (maybe direct dat a source calls?) this.contextSrv and this.contextSrv.user can never be null so those checks should not be needed (by looking at the code :))